admin/fmt

```diff
@@ -4,7 +4,13 @@ set -xe
 ADMIN=$(dirname "$0")
 PREV=$(pwd)
 if [ ! -d $ADMIN/rustfmt ] ; then
```

Comment:
It seems like we need to check the actual version of rustfmt stored in $ADMIN/rustfmt to see if it is the version that corresponds to the given commit. In particular, when we update $COMMIT below, this should trigger a re-download and rebuild of the new version of rustfmt.

Reply:
Please excuse this annoying comment: Bazel would do this as part of its approach. If we're going this route, it might be better to download Bazel and run this script through it, instead of depending on these shells scripts to get this right.